common/v2/services/Store/Network/helpers.ts

```diff
@@ -11,9 +11,14 @@ import {
   NodeOptions,
   WalletName
 } from 'v2/types';
+import { makeExplorer } from 'utils/helpers';
 
 export const getAllNetworks = () => {
-  return Object.values(getCache().networks);
+  const networks = Object.values(getCache().networks);
+  networks.forEach(
+    n => (n.blockExplorer = n.blockExplorer ? makeExplorer(n.blockExplorer) : n.blockExplorer)
```

Comment:
1. Avoid changing an objects attributes in place. Prefer returning a copy with object destructing or `Object.assign`
```
...
return networks.map(({ blockExplorer, ...rest }) => ({
  ...rest,
   blockExplorer: blockExplorer ? makeExplorer(blockExplorer) : blockExplorer 
}));
```

2. networks blockExplorer are already statically defined, currently in `config/networks/static/reducer.ts` (change in progress). Do we really need to run makeExplorer again?

Reply:
See below